src/integration/common/a11y.js

```diff
@@ -1,10 +1,9 @@
-export default ({ skipToContentText, headlineText }) => {
-  [amp, canonical].forEach((page) => {
+export default ({ skipToContentText }) => {
+  [amp, canonical].forEach(page => {
```

Comment:
```suggestion
  [amp, canonical].forEach((page) => {
```
Fix for the eslint error causing jenkins to fail.